build-tools/scripts/BuildEverything.mk

```diff
@@ -90,50 +88,43 @@ jenkins: prepare leeroy $(ZIP_OUTPUT)
 leeroy: leeroy-all framework-assemblies opentk-jcw
 
 leeroy-all:
-	$(foreach conf, $(CONFIGURATIONS), \
-		$(call MSBUILD_BINLOG,leeroy-all,$(_SLN_BUILD),$(conf)) $(SOLUTION) /p:Configuration=$(conf) $(_MSBUILD_ARGS) && \
-		$(call CREATE_THIRD_PARTY_NOTICES,$(conf),bin/$(conf)/lib/xamarin.android/ThirdPartyNotices.txt,$(THIRD_PARTY_NOTICE_LICENSE_TYPE),True,False) && ) \
-	true
+	$(call MSBUILD_BINLOG,leeroy-all,$(_SLN_BUILD),$(CONFIGURATION)) $(SOLUTION) /p:Configuration=$(CONFIGURATION) $(_MSBUILD_ARGS) && \
```

Comment:
We could simplify `MSBUILD_BINLOG` to not take `$(CONFIGURATION)` as a parameter, and simply use it in the template directly.